demisto_sdk/commands/format/update_incidentfields.py

```diff
@@ -64,52 +65,48 @@ def format_marketplaces_field_of_aliases(self):
         the function will update the marketplaces in the fields mapped by the aliases to be XSOAR marketplace only.
         """
 
-        if not self.id_set_path:
-            logger.info(
-                '[yellow]Skipping "Aliases" formatting as id_set_path argument is missing[/yellow]'
-            )
-
         aliases = self.data.get("Aliases", {})
         if aliases:
+
+            if not self.graph:
+                logger.info(
+                    f"Skipping formatting of marketplaces field of aliases for {self.source_file} as the "
+                    f"no-graph argument was given."
+                )
+                return
+
             for (
-                alias_field,
-                alias_field_file_path,
+                alias_cli_name,
+                aliases_file_path,
+                alias_marketplaces,
             ) in self._get_incident_fields_by_aliases(aliases):
 
-                marketplaces = get_item_marketplaces(
-                    item_path=alias_field_file_path, item_data=alias_field
-                )
+                if len(alias_marketplaces) != 1 or alias_marketplaces[0] != "xsoar":
+                    # There will always be only one incident field in the list because the cliName is unique
```

Comment:
not sure I entirely understood these lines, but there can be multiple content items with the same ID (even multiple **incident fields** with the same ID but with different marketplaces/fromversion values).
The only field that is 100% unique per node, is the path.
let's talk about it next week :)